.github/workflows/integration_test.yaml

```diff
@@ -27,14 +27,30 @@ jobs:
       - name: Setup operator environment
         uses: charmed-kubernetes/actions-operator@main
         with:
-          provider: microk8s
-      - name: Enable microk8s registry
-        # Add a step to wait for the registry to become available after it has
-        # been deployed to avoid any issues with trying to push images to it
-        # before it is ready
+          provider: lxd
+      - name: Install microk8s
```

Comment:
Would recommend doing this earlier if possible so that microk8s can finish setup in the background whilst other installation steps occur, will reduce the overall time for the CI/CD